docs/tutorials/introduction.rst

```diff
@@ -67,12 +67,12 @@ function ``foo`` that simply returns ``"bar"``.
    :class: seealso
 
    If you need help writing a native module using Python's C API, we recommend
-   checking out the following resources.
-
-   - `The official Python C API documentation <https://docs.python.org/3/extending/index.html>`_
-   - `RealPython's "Building a Python C Extension Module" introductory tutorial <https://realpython.com/build-python-c-extension-module/>`_
-   - `pysheeet's "C Extensions" page <https://www.pythonsheets.com/notes/python-c-extensions.html>`_
-   - `pysheeet-kr's "Python C API cheatsheet" page <https://pysheeet-kr.readthedocs.io/ko/latest/notes/python-capi.html>`_
+   checking out the `Extending and Embedding the Python Interpreter
+   <https://docs.python.org/3/extending/index.html>`_ guide in the Python
```

Comment:
I wonder if we can use intersphinx syntax for this link.

Reply:
That works, made the change